AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1484,6 +1484,11 @@ oneplus.com,tickets.ua,woman.ru,elle.ru,marieclaire.ru,europaplus.ru,krossovki.c
 beautyinsider.ru###articleSubscribeBlock
 officelife.media##.subscribe-plate
 pinsknews.by#?#.sidebar_list > li.widget:has(> div.textwidget > form[action] strong:contains(Подпишитесь))
+politico.com#?#.below-article-section div.below-story-text:has(> aside > section > div.sign-up__column > div.sign-up__summary > a[href="https://www.politico.com/newsletters"])
+politico.com#?#.container div.story-text > aside:has(> section > div.sign-up__column > div.js-tealium-newsletter)
+politico.com#?#.container .container__slot > section:has(> div.sign-up__column > div.js-tealium-newsletter)
+politico.com##aside[class*="morning"][class*="esu"]
+politico.com##.morning-energy
```

Comment:
where these rules must be applied?

Reply:
for example
![Screenshot 2020-08-09 at 15 27 48](https://user-images.githubusercontent.com/18536328/89732084-063da500-da55-11ea-9b06-acd7302ad7e6.png)

Reply:
![Screenshot 2020-08-09 at 15 29 52](https://user-images.githubusercontent.com/18536328/89732107-35541680-da55-11ea-878b-e519e1036b89.png)

Reply:
![Screenshot 2020-08-09 at 15 35 06](https://user-images.githubusercontent.com/18536328/89732212-f8d4ea80-da55-11ea-92a0-404eb869552c.png)